### PR TITLE
Feature: CircleCI 2.0

### PR DESCRIPTION
**Pull request by pavolzbell**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    steps:
+      - checkout
+      - run: sudo apt-get install postgresql-client
+      - restore_cache: { keys: ['{{ checksum "Gemfile.lock" }}'] }
+      - run: bundle install --path vendor/bundle
+      - save_cache: { key: '{{ checksum "Gemfile.lock" }}', paths: [vendor/bundle] }
+      - run: dockerize -wait tcp://localhost:5432 -timeout 60s
+      - run: mv .circleci/database.yml config/database.yml
+      - run: bundle exec rake db:create db:structure:load --trace
+      - run: bundle exec rspec
```
Comment (jsuchal) [inline .circleci/config.yml]: Este nieco taketo nam tu chyba nie? https://docs.codeclimate.com/docs/circle-ci-test-coverage-example

Comment (pavolzbell) [reply .circleci/config.yml]: aha, pozriem na to

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+          RAILS_ENV: test
+
+      - image: circleci/postgres:9.6.8-alpine-ram
+        environment:
+          POSTGRES_USER: ubuntu
+          POSTGRES_PASSWORD: ''
+
+    steps:
+      - checkout
+      - run: curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./test-reporter
```
Comment (eronisko) [inline .circleci/config.yml]: Preco si toto vlastne presunul z gemu do CI?

Comment (pavolzbell) [reply .circleci/config.yml]: soon to be deprecated https://github.com/codeclimate/ruby-test-reporter
takto je to po novom https://github.com/codeclimate/test-reporter

celkovo konfiguracia pre Circle CI 2.0 je o dost ukecanejsia

**Pull request merged**